veadk/agent.py

```diff
@@ -294,13 +298,15 @@ async def run(
         final_output = ""
         for _prompt in prompt:
             message = types.Content(role="user", parts=[types.Part(text=_prompt)])
-            final_output = await self._run(runner, user_id, session_id, message, stream)
+            final_output = await self._run(
```

Comment:
why do you change these lines?

Reply:
the line break caused by the lint tool has been restored